.github/ISSUE_TEMPLATE/bug_template.md

```diff
@@ -7,7 +7,7 @@ labels: "type : bug"
 
 ## Issue
 
-Describe the issue you are facing. Show us the implementation: screenshots, GIFs, etc.
+Describe the issue you are facing. Show us the implementation: screenshots, GIF, etc.
```

Comment:
Let's follow the latest content of https://github.com/nimblehq/git-template
```suggestion
Describe the issue you are facing. Show us the implementation: screenshots, GIFs, etc.
```

The same for other GitHub templates